psycop/common/model_training_v2/metrics/binary_metrics/base_binary_metric.py

```diff
@@ -4,9 +4,7 @@
 
 from psycop.common.model_training_v2.metrics.base_metric import CalculatedMetric
 
-from ...presplit_preprocessing.polars_frame import PolarsFrame
-
 
 class BinaryMetric(Protocol):
-    def calculate(self, y_true: PolarsFrame, y_pred: pl.Series) -> CalculatedMetric:
+    def calculate(self, y_true: pl.Series, y_pred: pl.Series) -> CalculatedMetric:
```

Comment:
I dunno if this is possible, but can we do pl.Series[int]?

Some binary metrics will need a threshold as well (e.g. accuracy); not sure how to handle that. Perhaps not a concern if we only want to support AUROC.

Reply:
This will be a problem in BaseEvalDataset.calculate_metrics as well.